### DISCUSSION TITLE
feat: Add Edit and Delete Buttons to Model Selection Dropdown

### DISCUSSION THREAD

**Discussion by bannert1337** [open-webui/open-webui#2869] [General]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I have to manage many models in the chat's model selection dropdown. Currently, all models are displayed without any options to edit or delete them directly from the dropdown, which makes it cumbersome to manage and organize the models efficiently.

**Describe the solution you'd like**
I would like to have edit and delete buttons next to each model in the chat's model selection dropdown. This would allow users to quickly introduce changes to the model's settings or remove models that are no longer needed, improving the overall user experience and making model management more intuitive and efficient.

**Describe alternatives you've considered**
An alternative solution could be to have a separate model management interface where users can edit and delete models. However, this would require navigating away from the chat interface, which is less convenient than having the options directly in the dropdown.

**Additional context**
Adding these buttons would create shortcuts and improve the quality of life for users who frequently manage multiple models.

**Comment by justinh-rahb** [comment]
I think this would lead to clutter. Is there something wrong with the available options in Workspace > Models? You can use the three dot menu to hide individual models from the chat model picker list.

**Reply by bannert1337** [reply]
Well there are extra steps. You could go two routes:

Open Side-panel > Click Workspace > scroll to find model > click edit button

or

Open model selection dropdown > scroll to find model > click edit button

**Reply by justinh-rahb** [reply]
How often are you having to do this that we should clutter up the already crowded model list though? For one less click, I don't think the tradeoff is worth it.
